### PR TITLE
#161587535 Add responsiveness to comment deletion confirmation modal

### PR DESCRIPTION
**Pull request by Nedson202**
#### What does this PR do?
Updates responsiveness of comment deletion confirmation modal on mobile screens

#### Description of Task to be completed?
- update delete confirmation modal to be responsive on mobile screens

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Enter into the project directory - cd travel_tool_front && git checkout bg-delete-comment-responsiveness-161587535
- Install dependencies and start the app - make start
- Start the backend. git clone https://github.com/andela/travel_tool_back.git && make start
- Navigate to travela in your browser and log onto the app.
- Click the created request either on Approvals or Requests page
- Add comments
- Click on delete on comment header then resize to different mobile form factors

#### Any background context you want to provide?
The modal displays nicely on all screens available on chrome's responsive feature

#### What are the relevant pivotal tracker stories?
[#161587535](https://www.pivotaltracker.com/story/show/161587535)

#### Screenshots (if appropriate)
responsive

#### Questions:

### CODE REVIEW

**Comment by janetnim** [comment]
When I test on your branch this is what I see but am not sure that the delete modal is responsive. Am I missing something?
screen shot 2018-10-31 at 16 46 34

**Comment by Nedson202** [comment]
Let me look into this.

**Comment by janetnim** [comment]
Works just fine. Nice job.

File: src/components/RequestsModal/RequestsModal.scss [hunk 1]
```diff
@@ -334,5 +334,4 @@
   position: relative;
   top: -88px;
   right: -95px;
-}
-
+}
```
[the remark this answers is not in the document]
Comment (Nedson202) [reply src/components/RequestsModal/RequestsModal.scss]: Done.

**Pull request merged**